### DISCUSSION TITLE
Automatically print when the page finishes loading?

### DISCUSSION THREAD

**Discussion by jjhhrj** [Budibase/budibase#13369] [Help]
Hello, I have tried many ways to make an auto-print workflow when clicking on the record. But here are my problems.
Firstly, I've set an action when the record is clicked in, which means that when a record is clicked in, there will be a unique user ID presented
![image](https://github.com/Budibase/budibase/assets/164453240/cb6a92dc-429f-4fa7-8c6f-9321de622af5)
after that, on the page i want to print, I've input a embed code:
![image](https://github.com/Budibase/budibase/assets/164453240/458a73df-f299-4390-8fad-e670c4c506f6)
but the problem is printing dialog appears before the printing page completed loading, so I tried to set the time for it
![image](https://github.com/Budibase/budibase/assets/164453240/6c42e9fb-163f-414e-88dd-e0278ef0895f)
But it still didnt work. Anyone know how to fix this?

**Comment by ConorWebb96** [comment]
Closing as its a duplicate of #13354.

I believe you have asked this within the Discord as well. It was suggested that you use a custom plugin to achieve what you are after.
